src/main/java/me/zeroeightsix/kami/KamiMod.java

```diff
@@ -98,6 +99,7 @@
     public static boolean hasAskedToUpdate = false;
 
     public static TemporaryPauseProcess pauseProcess;
+    public static AutoObsidianProcess autoObsidianProcess;
```

Comment:
is there a good reason as to why this is in kamimod?

Reply:
Just to initialize the baritone process